packages/machinomy/src/util/expects_rejection.ts

```diff
@@ -1,6 +1,6 @@
 const expect = require('expect')
 
-export default function expectsRejection (res: Promise<any>): Promise<any> {
+export function expectsRejectionType<T = Error> (res: Promise<any>, constructor: {new (): T}): Promise<any> {
```

Comment:
Why? What does it bring?

Reply:
To verify that the error has a certain type. It will be useful in case we check what is expected to be allowed TransportVersionNotSupportError (not base Error)

Reply:
Just don't. Please do not introduce complexity before it is needed.